applications/asset_tracker_v2/doc/gnss_module.rst

```diff
@@ -0,0 +1,122 @@
+.. _asset_tracker_v2_gnss_module:
+
+GNSS module
+###########
+
+.. contents::
+   :local:
+   :depth: 2
+
+The GNSS module controls the GNSS functionality of the nRF9160.
+It can be used to acquire the location of the device, either in the form of events containing a position, velocity and time (PVT) structure or NMEA sentences.
```

Comment:
Maybe provide links to online docs or nordic docs on GGA, NMEA, and PVT? So that users can do a quick reference/doc lookup if they are curious.